test/ci/cpu_unit_tests_linux.yml

```diff
@@ -51,6 +51,13 @@ jobs:
       pytest test/unit/test_unified_explainer.py --junitxml=./TEST--TEST.xml -o junit_suite_name="$(Agent.JobName)"
     displayName: 'Run MSR Asia tests'
 
+  - bash: |
+      cd /usr/share/miniconda/bin
+      source activate interpret_cpu
+      cd /home/vsts/work/1/s
```

Comment:
why do you need to cd for this? I would prefer not depending on knowing where things are if possible